predicators/envs/base_env.py

```diff
@@ -305,6 +305,7 @@ def _set_seed(self, seed: int) -> None:
     def reset(self, train_or_test: str, task_idx: int) -> State:
```

Comment:
this and the other `State` types here will soon be replaced by `Observation`, but I'm deferring that until the CogMan PR